__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const app = require('../app.js');
 const db = require('../db/connection.js');
 const seed = require('../db/seeds/seed.js');
 const data = require('../db/data/development-data/');
+const { response } = require('../app.js');
```

Comment:
not sure if this is an auto import or not?

Reply:
Think it was. I hate it when that happens! 
Removed it now.